js/modules/tamagotchi.js

```diff
@@ -1,18 +1,64 @@
 export default class Tamagotchi {
-  constructor() {
-    this.health = { value: 10, importance: 1 };
-    this.hunger = { value: 10, importance: 3 };
-    this.energy = { value: 10, importance: 2 };
-    this.fun = { value: 10, importance: 4 };
-    console.log("Tamagotchi initialized");
-  }
+	constructor() {
+		this.health = { value: 10, importance: 1 };
+		this.hunger = { value: 10, importance: 3 };
+		this.energy = { value: 10, importance: 2 };
+		this.fun = { value: 10, importance: 4 };
+		console.log('Tamagotchi initialized');
+	}
 
-  displayHealth = (elementSelector) => {
-    const displayElement = document.querySelector(elementSelector);
-    displayElement.innerText = this.energy.value;
-  };
+	displayHealth = elementSelector => {
+		const displayElement = document.querySelector(elementSelector);
+		displayElement.innerText = this.health.value;
+	};
+	displayHunger = elementSelector => {
+		const displayElement = document.querySelector(elementSelector);
+		displayElement.innerText = this.hunger.value;
+	};
+	displayEnergy = elementSelector => {
+		const displayElement = document.querySelector(elementSelector);
+		displayElement.innerText = Math.floor(this.energy.value);
+	};
+	displayFun = elementSelector => {
+		const displayElement = document.querySelector(elementSelector);
+		displayElement.innerText = this.fun.value;
+	};
 
-  mount = ({ healthElement, hungerElement, energyElement, funElement }) => {
-    this.displayHealth(healthElement);
-  };
+	decreaseValues = () => {
+		this.hunger.value -= 1;
+		this.fun.value -= 1;
+
+		setTimeout(() => {
+			if (this.fun.value > 5) {
+				this.energy.value -= 0.5;
+			} else {
```

Comment:
**nitpick:** Would be good idea to split it into smaller functions, like `decreaseHealth` or `decreaseEnergy`, so you can have better control over these conditions, because each of life parameter has different conditions